pybricks/hubs/pb_type_movehub.c

```diff
@@ -185,7 +185,7 @@ STATIC mp_obj_t hubs_MoveHub_IMU_acceleration(mp_obj_t self_in) {
     // Convert to appropriate units and return as tuple
     mp_obj_t values[3];
     for (uint8_t i = 0; i < 3; i++) {
-        values[i] = MP_OBJ_NEW_SMALL_INT((data[i] * 10) >> 6);
+        values[i] = MP_OBJ_NEW_SMALL_INT(((data[i] + 1) * 11) >> 3);
```

Comment:
Would this change the unit to mm/s/s, then?

```suggestion
        values[i] = MP_OBJ_NEW_SMALL_INT((data[i] * 10000) >> 6);
```

Reply:
It looks right, assuming the existing code was correct.

You could test and see if it reads something close to 9800 if you compile and run the change.

Reply:
Looking at the [datasheet](https://www.st.com/resource/en/datasheet/lis3dh.pdf), it looks like the sensor returns a value in *mg* so if the value is closer to 10000, then we may need to use `* 98000` instead.

Reply:
It reads `10313`